.gitignore

```diff
@@ -145,3 +145,6 @@ make
 # CLion
 .idea
 cmake-build-debug
+
+# Output from running db4 installation
+db4/
```

Comment:
at best we should mark 14515 as partial. IMO we should drop since we don't support openbsd builds